frontend/javascripts/oxalis/view/left-border-tabs/histogram_view.js

```diff
@@ -209,6 +291,18 @@ class Histogram extends React.PureComponent<HistogramProps, HistogramState> {
           width={canvasWidth}
           height={canvasHeight}
         />
+        <Tooltip title={tooltipText}>
+          <VerticalAlignMiddleOutlined
+            style={{
+              position: "absolute",
+              transform: "rotate(90deg)",
+              top: 14,
+              right: 82,
```

Comment:
It's a bit unusual to position the icon that way, but I see, that there's not really a super straightforward way to do it differently, since the button would need to be defined higher in the component hierarchy while still interacting with the histogram. This is a symptom of our UI being not super consistent (the button is only specific to the histogram view, but exists outside of it). Tldr: Let's keep it as is, but we should probably change it at some point in the future :tm: :)